components/card/README.md

```diff
@@ -1,30 +1,137 @@
+    <d2l-card-footer-link id="assignmentsLink1" icon="tier1:assignments" text="Assignments" secondary-text="1"></d2l-card-footer-link>
+    <d2l-tooltip position="top" style="width: 100%;" for="assignmentsLink1">You have 1 assignments due tomorrow.</d2l-tooltip>
+  </div>
+</d2l-card>
+```
+
+## Best Practices
+
+<!-- docs: start best practices -->
+<!-- docs: start dos -->
+* Roll up and emphasize important information to make it easy for users to find
+* Use cards when the user will benefit from a visual representation of the associated item
+* Use cards when the content doesn’t rely heavily on the sort order
+<!-- docs: end dos -->
+
+<!-- docs: start donts -->
+* Don't force the user to click in and out of cards to find information that’s important to the work flow
+* Don't use cards
+  * when the user needs to easily compare data from one card to another
+  * when the sort order needs to be emphasized – consider a list or table
+  * for user generated content
+<!-- docs: end donts -->
+<!-- docs: end best practices -->
+
+## Composition
+
+Cards are composed of 3 sections. Each card will have a content section but is not required to have a header or footer.
+
+<!-- docs: demo size:large -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/card/card.js';
+  import '@brightspace-ui/core/components/tooltip/tooltip.js';
+  import '@brightspace-ui/core/components/card/card-footer-link.js';
+</script>
+
+<style>
+  .container {
+    display: flex;
+  }
+  .title_container {
+    margin-right: 50px;
+  }
+  .title_container > div {
+    align-items: center;
+    display: flex;
+    justify-content: flex-end;
+  }
+</style>
+
+<div class="container">
+  <div class="title_container">
+    <div id="header" class="active" style="height: 155px;">
+      Header (optional)
+    </div>
+    <div id="content" style="height: 115px;">
+      Content (required)
+    </div>
+    <div id="footer">
+      Footer (optional)
+    </div>
+  </div>
+  <d2l-card align-center text="Biology" align-center style="height: 320px; width: 245px;">
+    <img slot="header" alt="" style="display: block; width: 100%;" src="https://s.brightspace.com/course-images/images/7905e442-f009-46f6-8586-2c273a7c0158/banner-narrow-low-density-max-size.jpg">
+    <div slot="content">
+      <div>Biology</div>
+      <d2l-card-content-meta>Science • Grade 11</d2l-card-content-meta>
+    </div>
+    <div slot="footer">
+      <d2l-card-footer-link id="discussionsLink1" icon="tier1:outcomes" text="Outcomes" secondary-text="2"></d2l-card-footer-link>
+      <d2l-tooltip for="discussionsLink1">Discussions</d2l-tooltip>
+      <d2l-card-footer-link id="assignmentsLink1" icon="tier1:assignments" text="Assignments" secondary-text="1"></d2l-card-footer-link>
+      <d2l-tooltip position="top" style="width: 100%;" for="assignmentsLink1">You have 1 assignments due tomorrow.</d2l-tooltip>
+    </div>
+  </d2l-card>
+</div>
+```
+
+## Card <d2l-card>
```

Comment:
Just noticed the `<>` around the tags - I think they still have to be square brackets instead